client/src/app/services/import.service.ts

```diff
@@ -13,8 +13,34 @@ export class ImportService {
 
   constructor(private http: HttpClient) { }
 
-  async importAccounting(profileId: Profile["id"], formData: FormData) {
-    return this.http.post(this.root + "/profiles/" + profileId + "/accounting", formData, { responseType: 'text' }).toPromise();
+  async importAccounting(profileId: Profile["id"], formData: FormData, isAppend?: boolean) {
```

Comment:
Too much repeating code. Maybe a common method import that all the other methods would call?

Reply:
Agreed; method created.